iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Branch.java

```diff
@@ -28,7 +31,7 @@
          * In particular, it determines the duration during which
          * the current current value may be sustained.
          */
-        CurrentLimits.TemporaryLimit getTemporaryLimit();
+        LoadingLimits.TemporaryLimit getTemporaryLimit();
```

Comment:
Should we be more specific? The type of the limit is not CURRENT anymore?

Reply:
Good remark... Not sure but I think for now, it will only be for current limits, I will keep it at that.